server/src/services/apps.service.ts

```diff
@@ -103,9 +97,8 @@ export class AppsService {
     });
   }
 
-  async create(user: User, manager: EntityManager): Promise<App> {
+  async create(user: User, manager: EntityManager, name: string): Promise<App> {
```

Comment:
```suggestion
  async create(name: string, user: User, manager: EntityManager): Promise<App> {
```
lets change the arguments order. some case entity manager can be empty

Reply:
done.